qemu/tests/cfg/hv_enforce_cpuid_msr_check.cfg

```diff
@@ -0,0 +1,21 @@
+- hv_enforce_cpuid_msr_check:
+    only Linux
+    type = hv_enforce_cpuid_msr_check
+    modprobe_cmd = "modprobe msr"
+    rdmsr_cmd = "rdmsr 0x40000002"
+    not_preprocess = yes
+    required_qemu = [6.2.0,)
+    RHEL.9:
+        repo_install_cmd = "yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm"
+    RHEL.8:
+        repo_install_cmd = "yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm"
+    RHEL.7:
+        repo_install_cmd = "yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm"
+    msr_install_cmd = "yum install -y msr-tools"
+    modify_submanager_cmd = "sed -i -e 's/enabled=1/enabled=0/' /etc/yum/pluginconf.d/subscription-manager.conf"
+    restore_submanager_cmd = "sed -i -e 's/enabled=0/enabled=1/' /etc/yum/pluginconf.d/subscription-manager.conf"
```

Comment:
hi @nickzhq , I tried to install msr-tools, and found that we don't have to disable subscription-manager, msr-tool was installed successfully, would you pls make a double check for this? We can remove these if no need to do the change.